framework/core/locale/core.yml

```diff
@@ -350,7 +350,7 @@ core:
     forgot_password:
       dismiss_button: => core.ref.okay
       email_placeholder: => core.ref.email
-      email_sent_message: We've sent you an email containing a link to reset your password. Check your spam folder if you don't receive it within the next minute or two.
+      email_sent_message: If this email is registered, we'll send an email containing a link to reset their password. Check your spam folder if you don't receive it within the next minute or two.
```

Comment:
```suggestion
      email_sent_message: If the email you entered is registered with this site, we'll send you an email containing a link to reset your password. Check your spam folder if you don't receive it within the next minute or two.
```

Reply:
Minor wording nit: wanted to expand on what "registered" means, which email we're talking about, and a consistent "you"/"your" usage instead of "their".